master_server/Request.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
```

Comment:
I have left comments on proto formatting in Neeraj's change. Please follow the changes he will make.

Reply:
Changes have been made as suggested.